service/src/main/kotlin/fi/espoo/evaka/messaging/MessageService.kt

```diff
@@ -26,6 +26,17 @@ class MessageService(
     private val notificationEmailService: MessageNotificationEmailService
 ) {
     init {
+        asyncJobRunner.registerHandler {
+            db: Database.Connection,
+            _: EvakaClock,
+            msg: UrgentAsyncJob.MarkMessageAsSent ->
+            db.transaction {
+                it.upsertReceiverThreadParticipants(msg.threadId, msg.recipientIds, msg.sentAt)
+                it.markMessagesAsSent(msg.messageIds, msg.sentAt)
+            }
+        }
```

Comment:
We need to add one more field to the job payload. Is it better to add a new `AsyncJob` or add a nullable parameter to the existing one? 🤷